softgroup/services/profile/profile-api/src/main/java/com/softgroup/profile/api/message/GetContactProfilesResponse.java

```diff
@@ -12,13 +13,13 @@
  * Time: 10:09
  */
 public class GetContactProfilesResponse implements ResponseData {
-    private List<ProfileEntity> profileEntities;
+    private List<ContactEntity> contactEntities;
```

Comment:
@VadymPolyanski Omg. Entity... What I've told about entities ? And next - this is now what is written in API